lib/src/test/java/vmath/Mat4Test.java

```diff
@@ -2,10 +2,26 @@
 
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 public class Mat4Test {
 
     @Test
     void givenNoArgs_whenMatrix4Created_thenMatrixIsIdentity() {
-        //TODO(): check default constructor for identity matrix using get() method
+        Mat4 m = new Mat4();
+
+        assertEquals(m.get(3,2), 0.0);
+        assertEquals(m.get(0,0), 1.0);
+        assertEquals(m.get(3,3), 1.0);
+    }
```

Comment:
This test does not check all indices of the matrix, and therefore it is possible that it could succeed when the matrix is not an identity matrix. Please edit the test to check the whole matrix so that the test does what its name says.

Reply:
Added a test to check all indices of the matrix.